presto-main/src/main/java/com/facebook/presto/server/QuerySessionSupplier.java

```diff
@@ -55,6 +56,7 @@
     private static final File SESSION_PROPERTY_CONFIGURATION = new File("etc/session-property-config.properties");
     private static final String SESSION_PROPERTY_MANAGER_NAME = "session-property-config.configuration-manager";
 
+    private final SessionPropertyConfigurationManagerContextInstance configurationManagerContext;
```

Comment:
nit: We can use `SessionPropertyConfigurationManagerContext` (the interface) as the type here.